index.js

```diff
@@ -653,6 +653,9 @@ async function sendBatteryAlert(locationid, signal) {
 
 const server = app.listen(8080);
 helpers.log('brave server listening on port 8080')
+if(!helpers.isTestEnvironment()){
+    setInterval(checkHeartbeat, 1000)
```

Comment:
Is this once a second? That seems extreme.

Reply:
yeah. The actual alert threshold is 60 seconds right now, we could do the same for the interval that we check at? or 30 seconds, any suggestion?

this is also the same interval as for Buttons

Reply:
Ah! That makes sense about the threshold being different than how often we check.

I guess we don't want the maximum time it could be down to be longer than our stillness threshold (which is 60 seconds, right?). So that's probably why we did it this way. What if we did this check every 15 seconds and changed the alert threshold to 45 seconds?

Good point about Buttons. Maybe it's overkill there too, but it does mean that we get more accurate numbers for `heartbeatDashboard` and we have fewer things to check on than we're hopefully going to have for Sensors.

It's up to you :smile: